dom.bs

```diff
@@ -4602,15 +4679,24 @@ invoked, must run these steps:
  <a>converted to ASCII lowercase</a>.
  <!-- XXX why restrict this to HTML documents? -->
 
- <li>Let <var>interface</var> be the
- <a>element interface</a> for
- <var>localName</var> and the <a>HTML namespace</a>.
+ <li>Let <var>is</var> be the value of <code>is</code> member of <var>options</var>, or null if no
+ such member exists.
+
+ <li>Let <var>definition</var> be the result of
+ <a lt="look up a custom element definition">looking up a custom element definition</a>, given the
+ <a>context object</a>, the <a>HTML namespace</a>, <var>localName</var>, and <var>is</var>.
+
+ <li>If <var>is</var> is non-null and <var>definition</var> is null, then <a>throw</a> a
+ {{NotFoundError}}.
 
- <li>Return a new <a for="/">element</a> that implements <var>interface</var>,
- with no attributes,
- <a for=Element>namespace</a> set to the <a>HTML namespace</a>,
- <a for=Element>local name</a> set to <var>localName</var>, and
- <a>node document</a> set to the <a>context object</a>.
+ <li>Let <var>element</var> be the result of <a>creating an element</a> given the
+ <a>context object</a>, <var>localName</var>, null, the <a>HTML namespace</a>, <var>is</var>, and
+ with the <var>synchronous custom elements</var> flag set. Rethrow any exceptions.
+
+ <li>If <var>is</var> is non-null, then <a>set an attribute value</a> for <var>element</var> using
+ "<code>is</code>" and <var>is</var>.
```

Comment:
This means you get an `attributeChangedCallback` for `is`, right? And queued mutation observers if you had set those up on yourself. That's what we want?